sdk/cosmos/microsoft-azure-cosmos-benchmark/src/main/java/com/azure/data/cosmos/benchmark/Configuration.java

```diff
@@ -55,7 +55,7 @@
     private int documentDataFieldSize = 20;
 
     @Parameter(names = "-maxConnectionPoolSize", description = "Max Connection Pool Size")
-    private Integer maxConnectionPoolSize = 1000;
+    private Integer maxConnectionPoolSize = 500;
```

Comment:
we have many different endpoint in the direct mode.

with `maxConnectionPoolSize = 500;` does it mean the connection pool size to each endpoint is at most 500 or does it mean total number of connections to all endpoints at most is 500?

as a separate question why are we changing this to 500?

Reply:
It is at most 500 concurrent connections (combining all endpoints). 

Reactor Netty has done some performance benchmarking, and they suggest 500 as max connection pool size. 
Can't find the article right now, will post it here once I find it.

Reply:
Changed it back to 1000 connections.